### DISCUSSION TITLE
Release 1.1

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#119] [General]
# This is the last 1.X release
### Thank you everyone for the donations to support the project! 
While these are far from mandatory, it proves to me the high interest about this project and brings extra motivation to continue! 

> ### Initially I've planned more for the 1.1 release, but after some consideration I think it's best to include all the new features in the next increment (2.x). So after quite some bug-fixes and QOL improvements, I've decided that it's time to release the 1.1 update.

## What's Changed (PRs)
* new icon by @T8RIN in https://github.com/IacobIonut01/Gallery/pull/59
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/49
* Configure Renovate by @renovate in https://github.com/IacobIonut01/Gallery/pull/80
* Update kotlinCoroutinesVersion to v1.7.2 by @renovate in https://github.com/IacobIonut01/Gallery/pull/89
* Update room to v2.5.2 by @renovate in https://github.com/IacobIonut01/Gallery/pull/91
* Update dependency androidx.compose:compose-bom to v2023.06.01 by @renovate in https://github.com/IacobIonut01/Gallery/pull/86
* Update dependency gradle to v8.2 by @renovate in https://github.com/IacobIonut01/Gallery/pull/94
* Update material3 to v1.2.0-alpha03 by @renovate in https://github.com/IacobIonut01/Gallery/pull/93
* Update agp to v8.2.0-alpha10 by @renovate in https://github.com/IacobIonut01/Gallery/pull/81
* Update plugin baselineProfilePlugin to v1.2.0-alpha16 by @renovate in https://github.com/IacobIonut01/Gallery/pull/90
* Update dependency composeOptions to v1.4.8 by @renovate in https://github.com/IacobIonut01/Gallery/pull/87
* Update dependency org.jetbrains.kotlin.android to v1.8.22 by @renovate in https://github.com/IacobIonut01/Gallery/pull/88
* Update dependency androidx.benchmark:benchmark-macro-junit4 to v1.2.0-alpha16 by @renovate in https://github.com/IacobIonut01/Gallery/pull/82
* Update actions/setup-java action to v3 by @renovate in https://github.com/IacobIonut01/Gallery/pull/85
* Update actions/checkout action to v3 by @renovate in https://github.com/IacobIonut01/Gallery/pull/84
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/73
* Update dependency androidx.media3:media3-ui to v1.1.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/97
* Update dependency org.jetbrains.kotlin.android to v1.9.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/98
* Update dependency androidx.media3:media3-exoplayer to v1.1.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/96
* Revert "Update dependency org.jetbrains.kotlin.android to v1.9.0" by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/101
* fix image click by @lokomoto in https://github.com/IacobIonut01/Gallery/pull/99
* Update agp to v8.2.0-alpha11 by @renovate in https://github.com/IacobIonut01/Gallery/pull/100
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/104
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/118

## New Contributors
* @T8RIN made their first contribution in https://github.com/IacobIonut01/Gallery/pull/59
* @renovate made their first contribution in https://github.com/IacobIonut01/Gallery/pull/80
* @lokomoto made their first contribution in https://github.com/IacobIonut01/Gallery/pull/99

## **Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/v1.0.4...v1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/v1.1'>Release 1.1</a>.</em>

**Comment by whyakari** [comment]
the app is crashing and exiting when I click delete some photo.

**Reply by IacobIonut01** [reply]
Create an issue and add more details about the crash
